Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM --platform=$BUILDPLATFORM eclipse-temurin:19-jre-jammy AS builder
+FROM --platform=$BUILDPLATFORM eclipse-temurin:19-jdk-jammy AS builder
```

Comment:
I had to change the `Dockerfile` to use a JDK as the base image (rather than a JRE)
This is because we are using `mapstruct`, which in turn uses `kapt`, which in turn requires a JDK at build time!
Hopefully we wont get frowned upon for doing this .... 😢